op_crates/web/00_dom_exception.js

```diff
@@ -1,15 +1,47 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-class DOMException extends Error {
-  #name = "";
+((window) => {
+  const nameToCodeMapping = Object.create(null);
+  nameToCodeMatting.IndexSizeError = 1;
+  nameToCodeMapping.HierarchyRequestError = 3;
+  nameToCodeMapping.WrongDocumentError = 4;
+  nameToCodeMapping.InvalidCharacterError = 5;
+  nameToCodeMapping.NoModificationAllowedError = 7;
+  nameToCodeMapping.NotFoundError = 8;
+  nameToCodeMapping.NotSupportedError = 9;
+  nameToCodeMapping.InvalidStateError = 11;
+  nameToCodeMapping.SyntaxError = 12;
+  nameToCodeMapping.InvalidModificationError = 13;
+  nameToCodeMapping.NamespaceError = 14;
+  nameToCodeMapping.InvalidAccessError = 15;
+  nameToCodeMapping.TypeMismatchError = 17;
+  nameToCodeMapping.SecurityError = 18;
+  nameToCodeMapping.NetworkError = 19;
+  nameToCodeMapping.AbortError = 20;
+  nameToCodeMapping.URLMismatchError = 21;
+  nameToCodeMapping.QuotaExceededError = 22;
+  nameToCodeMapping.TimeoutError = 23;
+  nameToCodeMapping.InvalidNodeTypeError = 24;
+  nameToCodeMapping.DataCloneError = 25;
```

Comment:
There was a spelling error `nameToCodeMatting` => `nameToCodeMapping`
But this can be a single call anyway:
```suggestion
const nameToCodeMapping = Object.create(
  null,
  {
    IndexSizeError: { value: 1 },
    HierarchyRequestError: { value: 3 },
    WrongDocumentError: { value: 4 },
    InvalidCharacterError: { value: 5 },
    NoModificationAllowedError: { value: 7 },
    NotFoundError: { value: 8 },
    NotSupportedError: { value: 9 },
    InvalidStateError: { value: 11 },
    SyntaxError: { value: 12 },
    InvalidModificationError: { value: 13 },
    NamespaceError: { value: 14 },
    InvalidAccessError: { value: 15 },
    TypeMismatchError: { value: 17 },
    SecurityError: { value: 18 },
    NetworkError: { value: 19 },
    AbortError: { value: 20 },
    URLMismatchError: { value: 21 },
    QuotaExceededError: { value: 22 },
    TimeoutError: { value: 23 },
    InvalidNodeTypeError: { value: 24 },
    DataCloneError: { value: 25 },
  },
);
```

Reply:
1. Why wrap in an object with value?
2. Yes, one can also use `Object.assign`. There are many ways to do the same thing. However I proposed writing assignments and @kitsonk approved.

Reply:
I didn't know about the second argument. Thanks.